packages/schematics/angular/config/index.ts

```diff
@@ -103,3 +106,53 @@ function addKarmaConfig(options: ConfigOptions): Rule {
     );
   });
 }
+
+function addAiContextFile(options: ConfigOptions): Rule {
+  let fileName: string;
+  let filePath: string;
+  switch (options.tool) {
+    case 'gemini':
+      fileName = 'GEMINI.md';
+      filePath = '.gemini';
+      break;
+    case 'copilot':
+      fileName = 'copilot-instructions.md';
+      filePath = '.github';
+      break;
+    case 'claude':
+      fileName = 'CLAUDE.md';
+      filePath = '.claude';
+      break;
+    case 'cursor':
+      fileName = 'cursor.md';
+      filePath = path.join('.cursor', 'rules');
+      break;
+    case 'windsurf':
+      fileName = 'guidelines.md';
+      filePath = path.join('.windsurf', 'rules');
+      break;
+    default:
+      throw new SchematicsException(`Unsupported AI tool: ${options.tool}`);
```

Comment:
This is redundant as the validation is handed already during the schema validation.